spec/spec.md

```diff
@@ -87,18 +97,34 @@ Presentation Definitions are objects generate to articulate what proofs an entit
         "purpose": "We need your bank and account information."
       },
       "constraints": {
+        // TODO: how do we say that the prover must have a certain relationship to
+        // the credential subject (e.g., BE the one and only subject, but
+        // also things like "be the secondary account holder" or be the
+        // father on the birth certificate?
```

Comment:
ah now I understand prover. We had brought this up with dogs and their owners https://github.com/decentralized-identity/presentation-exchange/issues/12

Reply:
also discussed in issue #24